vendor/desktop-trampoline/test/desktop-trampoline.test.js

```diff
@@ -0,0 +1,153 @@
+const { stat, access } = require('fs').promises
```

Comment:
I think I will rewrite these tests to use node's test runner in a different PR (and after I've learned from @niik 's work 😂 ) if that's ok!